<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSettingsVisualization</class>
 <widget class="QWidget" name="DialogSettingsVisualization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Vertices Size</string>
       </property>
       <property name="buddy">
        <cstring>m_verticesSize</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QSpinBox" name="m_verticesSize">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Edges width</string>
       </property>
       <property name="buddy">
        <cstring>m_edgesWidth</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="m_edgesWidth">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2" rowspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>9</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Screen&amp;shot directory</string>
     </property>
     <property name="buddy">
      <cstring>m_screenshotDirectory</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="m_screenshotDirectory"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_browseButton">
       <property name="text">
        <string>&amp;Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>9</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_verticesSize</tabstop>
  <tabstop>m_edgesWidth</tabstop>
  <tabstop>m_screenshotDirectory</tabstop>
  <tabstop>m_browseButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
